### PR TITLE
Setting default throttle class to ServiceUserThrottle

### PR DESCRIPTION
**Pull request by madnihamza1841**
**Description:** Due to the limit on user throttle set to 75/minute in default UserRateThrottle class, the data loader was failing on sending more requests per minute. The default class was changed to ServiceUserThrottle to allow any number of requests per minute for Ecommerce Service worker.

**JIRA:** https://edlyio.atlassian.net/browse/EDLY-4260

**Testing Instructions:**
1. Run the data loader on Production. 
2. It should complete without giving any error.

### CODE REVIEW

File: ecommerce/extensions/api/throttles.py [hunk 1]
```diff
@@ -11,7 +11,11 @@ def allow_request(self, request, view):
         """Returns True if the request is coming from one of the service users
         and defaults to UserRateThrottle's configured setting otherwise.
         """
-        service_users = [settings.ECOMMERCE_SERVICE_WORKER_USERNAME, settings.PROSPECTUS_WORKER_USERNAME]
+        service_users = [
+            settings.ECOMMERCE_SERVICE_WORKER_USERNAME, 
+            settings.PROSPECTUS_WORKER_USERNAME, 
+            settings.DISCOVERY_SERVICE_WORKER_USERNAME
```
Comment (taimoor-ahmed-1) [inline ecommerce/extensions/api/throttles.py:17]: add comma to reduce git diff

**Review by taimoor-ahmed-1 — COMMENTED** [review]
minor suggestions

**Review by burhandodhy — APPROVED** [review]
LGTM

**Review by taimoor-ahmed-1 — APPROVED** [review]
LGTM

**Pull request merged**